Dockerfile

```diff
@@ -1,15 +1,33 @@
-FROM node:6.11-alpine
+FROM node:8.2.1-alpine

```

Comment:
Now that [Issue 16766](https://github.com/npm/npm/issues/16766) & [Issue 16892](https://github.com/npm/npm/issues/16892) are closed, we can use node 8.2.1 with Web Apps on linux 👍